article for claude image gen (#71)
ragTechDev/ragtech-website: data/posts/2026-04-10-claude-image-generation/index.md

---
title: "Why I Stopped Using AI Image Generation"
slug: "claude-image-generation"
author:
  name: "Saloni Kaur"
  profilePicture: "/assets/team/saloni.PNG"
publishedAt: "2026-04-10"
coverImage: "/posts/2026-04-10-claude-image-generation/cover-image.png"
brief: "I haven't used AI image generation tools in over a year. Here's why: privacy, energy, and what Anthropic says about why Claude doesn't generate images at all."
tags: ["AI", "image generation", "privacy", "Claude", "Anthropic", "energy", "AI ethics", "responsible AI"]
status: "published"
topic:
  - "ragTech"
readTimeInMinutes: 5
newsletter:
  send: true
  sent: false
  topic:
    - "ragTech"
seo:
  metaDescription: "Why one tech podcaster stopped using AI image generation tools — covering privacy risks, energy consumption, and Anthropic's stance on why Claude refuses to generate images."
  keywords: ["AI image generation", "Claude no image generation", "Anthropic image policy", "AI privacy", "AI energy consumption", "AI deep fakes", "responsible AI", "Claude safety"]
instagramEmbeds:
tiktokEmbeds:
---

> It's been well over a year since I last used an AI image generation tool. Not Midjourney, not DALL·E, not chatGPT.

Reviewer: ```suggestion
It's been well over a year since I last used an AI image generation tool. I haven't touched tools like Midjourney, DALL·E, or chatGPT's image generation feature in a *longgg* while.
```

# Why I Stopped Generating Images With AI

## Reason 1: Privacy

When you generate an image with an AI tool, you're not just typing a prompt into a blank void. You're interacting with a system that has been trained on images scraped from the internet (often without the explicit consent of the original creators). And when you submit a prompt, depending on the platform, that prompt (and sometimes the generated output) may be logged, stored, and used to further train or improve the model.

![Studio Ghibli AI art trend](https://miro.medium.com/v2/resize:fit:640/format:webp/1*SwyWIlX8jShYhMgo_U3r-g.png)
__Image from [LumenXo — The Hidden Dangers of AI Image Trends](https://medium.com/@inc.lumenxo/the-hidden-dangers-of-ai-image-trends-why-your-personal-photos-arent-safe-15daa2eafe45)__

> A recent and very visible example: remember the [Studio Ghibli AI art trend](https://www.business-standard.com/india-news/studio-ghibli-ai-art-trend-a-privacy-nightmare-in-disguise-experts-warn-125040600343_1.html) that swept the internet in early 2025? Millions of people uploaded their own photos to get them rendered in that iconic animation style. It was adorable. But cybersecurity experts quickly flagged what most participants didn't realise many of these platforms were collecting **biometric data**: facial structure, skin tone, expression patterns. The terms of service were often vague, with fine print granting platforms perpetual, royalty-free rights to use or modify your photos even after you deleted the app.

Reviewer: ```suggestion
A recent and very visible example: remember the [Studio Ghibli AI art trend](https://www.business-standard.com/india-news/studio-ghibli-ai-art-trend-a-privacy-nightmare-in-disguise-experts-warn-125040600343_1.html) that swept the internet in early 2025? Millions of people uploaded their own photos to get them rendered in that iconic animation style. It was adorable. But cybersecurity experts quickly flagged what most participants didn't realise: that many of these platforms were collecting **biometric data**: facial structure, skin tone, expression patterns. The terms of service were often vague, with fine print granting platforms perpetual, royalty-free rights to use or modify your photos even after you deleted the app.
```

And even beyond personal data, there's the question of whose work these models were trained on. Artists who never consented to having their style replicated. Photographers whose images were scraped and used without credit or compensation. The very act of generating an image using these tools is entangled in a web of unresolved ethical questions around data ownership and consent.

For me, that was enough to ask: *Do I really need this and want to support it?*

## Reason 2: Energy Consumption

The second reason is one that people don't talk about enough: **energy**.

Running an AI image generation model is computationally expensive. Unlike generating text, which is already energy-intensive, generating images requires significantly more GPU compute. Each image takes anywhere from seconds to minutes of sustained GPU activity, depending on the model and the resolution.

> [MIT Technology Review published research](https://www.technologyreview.com/2023/12/01/1084189/making-an-image-with-generative-ai-uses-as-much-energy-as-charging-your-phone/) showing that generating a single AI image consumes roughly the same energy as fully charging your smartphone. One image. One phone charge. And that same research found that image generation is **1,450 times more energy-intensive** than text models. That's not a marginal difference, a staggering one. A single image can release anywhere from 40 to 800 grams of CO₂ depending on the model used.

Reviewer: ```suggestion
[MIT Technology Review published research](https://www.technologyreview.com/2023/12/01/1084189/making-an-image-with-generative-ai-uses-as-much-energy-as-charging-your-phone/) showing that generating a single AI image consumes roughly the same energy as fully charging your smartphone. And that same research found that image generation is **1,450 times more energy-intensive** than text models. A single image can release anywhere from 40 to 800 grams of CO₂ depending on the model used.
```

That might sound trivial in isolation. But [MIT News estimates](https://news.mit.edu/2025/explained-generative-ai-environmental-impact-0117) that AI's annual carbon footprint could reach between 32.6 and 79.7 million tons of CO₂ and image generation is one of the most disproportionate contributors to that figure. Multiply one image's footprint across millions of daily generations, and the scale becomes hard to ignore.

![Explained: Generative AI’s environmental impact](https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/202501/MIT-AI-Climate-01_0.jpg?itok=kXVLyp49)
__Image from [MIT News](https://news.mit.edu/2025/explained-generative-ai-environmental-impact-0117)__

> We talk a lot about our personal carbon footprints. We swap out plastic straws, we bring reusable bags. But we rarely stop to think about the energy cost of the digital tools we use every day. AI image generation is, quietly, one of the more energy-hungry habits in the modern tech user's toolkit.

Reviewer: ```suggestion
We talk a lot about our personal carbon footprints. We swap out plastic straws, we bring reusable bags. But we rarely stop to think about the energy cost of the digital tools we use every day. AI image generation is, unbeknownst to most regular users, one of the more energy-hungry habits in the modern tech user's toolkit.
```

Since cutting it out, I've become much more deliberate about where I spend compute. It's a small change, but it's one I can live without.

# And Then There's Claude

> Here's the thing: when I started using Claude more heavily as my primary AI assistant, I noticed something. It doesn't generate images.

Reviewer: ```suggestion
When I started using Claude more heavily as my primary AI assistant, I noticed something. It doesn't generate images.
```
